lib/workers/send_user_invitation_email.ex

```diff
@@ -0,0 +1,46 @@
+defmodule ChatApi.Workers.SendUserInvitationEmail do
+  @moduledoc false
+
+  use Oban.Worker, queue: :mailers
+
+  alias ChatApi.{Accounts, Users}
+  alias ChatApi.Repo
+
+  require Logger
+
+  @impl Oban.Worker
+  @spec perform(Oban.Job.t()) :: :ok
+  def perform(%Oban.Job{
+        args: %{
+          "user_id" => user_id,
+          "account_id" => account_id,
+          "to_address" => to_address,
+          "invitation_token" => invitation_token
+        }
+      }) do
+    if send_user_invitation_email_enabled?() do
+      user = Users.find_by_id!(user_id) |> Repo.preload([:profile])
+      account = Accounts.get_account!(account_id)
+      Logger.info("Sending user invitation email to #{to_address}")
+
+      ChatApi.Emails.send_user_invitation_email(
+        user,
+        account,
+        to_address,
+        invitation_token
+      )
+    else
+      Logger.info("Skipping user invitation email to #{to_address}")
+    end
+
+    :ok
+  end
+
+  @spec send_user_invitation_email_enabled? :: boolean()
+  def send_user_invitation_email_enabled?() do
+    case System.get_env("USER_INVITATION_EMAIL_ENABLED") do
```

Comment:
nit: should we move this to `config/runtime.exs`, and we could also use `String.to_existing_atom` to cast boolean values. For example: https://github.com/plausible/analytics/blob/master/config/runtime.exs#L68

Reply:
I think that's a great idea, but it doesn't seem to be very standard practice in the codebase. We should have a larger discussion about how we should handle environmental variables. FWIW I agree that we should consolidate them into `config/runtime.exs`. 

cc @reichert621

Reply:
yup, let's handle that consolidation in a separate PR :+1: